Corona-Warn-App/src/main/java/de/rki/coronawarnapp/statistics/ui/stateselection/FederalStateSelectionFragment.kt

```diff
@@ -58,7 +58,14 @@ class FederalStateSelectionFragment : Fragment(R.layout.federal_state_selection_
             event.observe2(this@FederalStateSelectionFragment) {
                 when (it) {
                     FederalStateSelectionViewModel.Events.FinishEvent -> {
-                        findNavController().popBackStack(R.id.mainFragment, false)
+                        val navController = findNavController()
+                        navController.previousBackStackEntry?.savedStateHandle?.getLiveData(
+                            "comesFromAddLocalStats",
+                            true
+                        )
+                        navController.popBackStack(R.id.mainFragment, false).let {
+                            Bundle().apply { putBoolean("comesFromAddLocalStats", true) }
+                        }
                     }
```

Comment:
What is this for?

Reply:
Actually, this doesn't seem to do anything. I thought I needed it to announce the newly added statistics card, but it seems to work just fine without it. I've removed it.